hr_public_holidays/static/src/js/holidays_highlighter.js

```diff
@@ -0,0 +1,43 @@
+odoo.define('hr_public_holidays.holidays_highlighter', function (require) {
+'use strict';
+
+var calendarView = require('web_calendar.CalendarView');
+
+var Model = require('web.Model');
+var hrHolidays = new Model('hr.holidays.public.line');
+// tasked w/ fetching desired color from server params
+var irConfigParameter = new Model('ir.config_parameter');
+
+calendarView.include({
+    get_fc_init_options: function () {
+        var res = this._super();
+
+        var oldRenderHandler = res['viewRender'];
+        // extend view render callback w/ highlighting functionality
+        res['viewRender'] = function (view) {
+            oldRenderHandler.apply(this, arguments);  // read as super()
+            var visibleDays = $('.o_calendar_view .fc-day');
```

Comment:
this.$

Reply:
sadly enough, `TypeError: this.$ is not a function` :wink: